fiftyone/server/routes/embeddings.py

```
@@ -39,7 +41,16 @@ class OnPlotLoad(HTTPEndpoint):
     @route
     async def post(self, request: Request, data: dict) -> dict:
         """Loads an embeddings plot based on the current view."""
-        return await run_sync_task(self._post_sync, data)
+        try :
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)